Update GH label description in triaging process (#41035)
mdn/content: files/en-us/mdn/community/issues/index.md

Reviewer: Perhaps this
>     - Use the description field to indicate the status of the task or sub-tasks by using checklists.

should be flipped around
```
    - Use checklists in the description field to indicate the status of the task or sub-tasks.
```

Reviewer: ```suggestion
  - A discussion needs to take place to clarify the issue.
```

Reviewer: okay

### Before filing an issue

First, based on the type of problem you've discovered, identify the appropriate [MDN GitHub repository](/en-US/docs/MDN/Community/Our_repositories) to create the issue so the right people can handle it. Then, before creating an issue, check that it hasn't already been reported.

### Reporting an issue

Here are some hints for opening issues:

- Choose the appropriate category to report the issue. For example, to report a content bug, use the [Content issue](https://github.com/mdn/content/issues/new?assignees=&labels=needs+triage&template=content-bug.yml) template in the `mdn/content` repository.
- Provide sufficient information while reporting the issue:
  - **Issue title** must convey succinctly the _required action_.
  - **Issue description** must clearly describe the bug and, if you could, the action required to resolve the issue.

>     For complex issues that need multiple steps to resolve, break down the work into smaller tasks using a [task list](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-tasklists). Update the status of a task in the issue description instead of commenting on the issue. This helps others who may otherwise need to scroll through comments on the issue to determine the status of various tasks. Comments in an issue should be limited to details or context that help resolve the issue.

Reviewer: ```suggestion
    For complex issues that need multiple steps to resolve, break down the work into smaller tasks using a [task list](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-tasklists). Update the status of a task in the issue description instead of commenting on the issue. This helps others who may otherwise need to scroll through comments on the issue to determine the status of various tasks. Comments in an issue should be limited to details or context that help with resolving the issue.
```

- If you find yourself in one of the following situations, move the conversation to [MDN's discussion on GitHub](https://github.com/orgs/mdn/discussions):

>   - You are unsure if the issue is valid, or what the required action is.

Reviewer: ```suggestion
  - You are unsure if the issue is valid or what the required action is.
```

  - The requirements for completing the task expand while it's being resolved or the work is unclear.
- For minor bugs, you can [make the changes yourself](#fixing_issues_yourself) and submit a pull request.

> If the information you provide in the issue is incomplete, you might be asked to provide more details during the [issue triaging process](#review_the_issue_for_completeness_of_information).

Reviewer: Repeating the label info here as well in addition to that in the later section
```suggestion
If the information you provide in the issue is incomplete, you might be asked to provide more details during the [issue triaging process](#review_the_issue_for_completeness_of_information) (look out for `needs info` and `awaiting response` labels).
```

These are the general steps for working on an issue:

1. **Find an issue:** If you're looking to contribute, search for issues with the [`p2` or `p3`](#set_a_priority_label) label, or [one of the labels indicating accepting external PRs](#set_other_labels): `good first issue`, `accepting PR`, `help wanted`. Most repositories have issues with these labels. You are welcome to browse and pick an issue that is suitable for your skill set. Another useful place to look for issues to work on is the [MDN Contributors Task Board](https://github.com/orgs/mdn/projects/25). This project view lists open issues from multiple repositories. You can filter the list based on the topics (`Labels` column) you're interested in. See the description of some of the [labels](#set_other_labels) that get applied during the issue triage process.

   > [!NOTE]

>    > An issue with the `needs triage` label indicates that the MDN Web Docs core team has not reviewed the issue yet, and you shouldn't begin work on it.

Reviewer: ```suggestion
   > An issue with the `needs triage` label indicates that the core MDN team has not reviewed the issue yet, and you shouldn't begin work on it.
```

These are some of the things to keep in mind while reviewing the validity of an issue:

- Check if the issue raised is valid.
- Evaluate if the fix will improve the content for the readers and the website.
- Evaluate if the impact of the fix will be small or site-wide.
- Evaluate if the fix for the issue will need a discussion first, in which case, point the author to open a [discussion](https://github.com/orgs/mdn/discussions) instead.

> - Check if the issue complies with our [writing guidelines](/en-US/docs/MDN/Writing_guidelines/Writing_style_guide) and [templates](/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types).—for example, whether suggestions for adding links comply with our [external links policy](/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#external_links).

Author: ```suggestion
- Check if the issue complies with our [writing guidelines](/en-US/docs/MDN/Writing_guidelines/Writing_style_guide) and [templates](/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types). For example, whether suggestions for adding links comply with our [external links policy](/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#external_links).
```

After you've analyzed and validated the issue, and applied the appropriate labels (type, goal, effort, open to community), the triage process is complete and you can remove the `needs triage` label.

### Closing an issue as not planned

If an issue does not map to any action items, you should consider closing the issue as not planned. Click the drop down next to the **Close issue** button and select **Close as not planned**. Post a comment explaining the reason for closing the issue. Also add one of the following labels:

- `closed: browser bug`: The issue reports unexpected behavior (with their code or MDN's code) that's actually a browser bug, and it's not significant enough to warrant a BCD note.
- `closed: question`: The issue is based on a user error or misunderstanding, and the content is sufficiently correct and clear. However, usually do give some acknowledgment of the issue and consider whether the content can be clarified further to avoid similar questions.

> - `closed: wontfix`: The issue reports a valid "problem", but MDN decides not to fix it. Reasons include too much effort, out of scope, working as intended, etc.

Reviewer: ```suggestion
- `closed: browser bug`: The issue reports unexpected behavior in the reporter's code or in MDN's code, but the cause is a browser bug. The issue is also not significant enough to warrant a BCD note.
- `closed: question`: The issue is based on the reporter's error or misunderstanding, and the content is already correct and clear. However, you should still acknowledge the report and consider whether the content can be clarified further to prevent similar questions in the future.
- `closed: wontfix`: The issue reports a valid problem, but MDN decides not to fix it. Common reasons include the work being out of scope, the required effort outweighing the benefit, or content and code working as intended.
```

Next, set the following labels as appropriate:

- One of the problem type labels to indicate what needs to be fixed, if applicable:
  - `broken link external`: The issue reports a broken link to an external page.
  - `document not written`: The issue reports a necessary document that has not been written yet, usually because a link points to it. Note that we have a [Web Docs Backlog](https://openwebdocs.github.io/web-docs-backlog/all/) project that already tracks many unwritten reference pages, so if the issue is about one of those, you could close it and point to the backlog.
  - `screenshot`: The issue reports a missing or outdated screenshot.

>   - `baseline`: The issue reports an incorrect [baseline](/en-US/docs/Glossary/Baseline/Compatibility) banner. Note that this banner is rendered by a collaboration of BCD, frontend code, and the [web-platform-dx/web-features](https://github.com/web-platform-dx/web-features) project, but has nothing to do with content, so you should almost always either close the issue as working as intended, or transfer it somewhere else.

Author: A lot of the times it's easy to see why the baseline is rendered this way: for example, Chrome is not marked as "supported" because the feature is not supported on Chrome Android. There's nowhere to transfer it to because it's indeed correct. Other times when it's a BCD bug, it's very easy to see.
